data/openstudio_standards_data/database_files/level_2_electric_equipment.json

```diff
@@ -0,0 +1,1080 @@
+[
+    {
+        "electric_equipment_space_type_name": "corridor_all_other_electric_equipment",
+        "electric_equipment_minimum_epd": 0,
```

Comment:
what is the point of minimum, average, median, and maximum epd?

Reply:
This a sub-product of the bottom-up approach used to develop these values, providing a "range" instead of one single value. I'll let @yunyangye @yunyang-ye provide more info.